src/HealthGPS/healthgps.cpp

```diff
@@ -331,8 +332,13 @@ std::map<std::string, core::UnivariateSummary> HealthGPS::create_input_data_summ
     auto summary = std::map<std::string, core::UnivariateSummary>();
     auto &input_data = definition_.inputs().data();
     for (const auto &entry : context_.mapping()) {
-        input_data.column(entry.name()).accept(visitor);
-        summary.emplace(entry.name(), visitor.get_summary());
+        try {
+            input_data.column(entry.name()).accept(visitor);
+            summary.emplace(entry.name(), visitor.get_summary());
+        } catch (const std::out_of_range &oor) {
```

Comment:
Unused variable?

Reply:
Fixed in [e0d7b0a](https://github.com/imperialCHEPI/healthgps/pull/157/commits/e0d7b0ad1658d3f6d95b02ebe373c82e66d6c290)